### PR TITLE
Add SOS support

### PR DESCRIPTION
**Pull request by pangea-andrest**

### CODE REVIEW

**Review by kenany — DISMISSED** [review]

**Review by kenany — DISMISSED** [review]

**Review by kenany — DISMISSED** [review]

File: packages/pangea-sdk/pangea/asyncio/services/store.py [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+        path: Optional[str] = None,
+        add_metadata: Optional[m.Metadata] = None,
+        remove_metadata: Optional[m.Metadata] = None,
+        metadata: Optional[m.Metadata] = None,
+        add_tags: Optional[m.Tags] = None,
+        remove_tags: Optional[m.Tags] = None,
+        tags: Optional[m.Tags] = None,
+        parent_id: Optional[str] = None,
+        updated_at: Optional[str] = None,
+    ) -> PangeaResponse[m.UpdateResult]:
```
Comment (r3dcrosse) [inline packages/pangea-sdk/pangea/asyncio/services/store.py:170]: Missing param? See https://github.com/pangeacyber/pangea-go/pull/155#discussion_r1446641529

**Review by kenany — APPROVED** [review]

**Review by kenany — DISMISSED** [review]

File: examples/store/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+In the example root directory (./examples/store), run the following command:
+
+```
+poetry install
+```
+
+Set up the environment variables ([Instructions](https://pangea.cloud/docs/getting-started/integrate/#set-environment-variables)) `PANGEA_STORE_TOKEN` and `PANGEA_DOMAIN` with your project token configured on the Pangea User Console (token should have access to Store service [Instructions](https://pangea.cloud/docs/getting-started/configure-services/#configure-a-pangea-service)) and with your Pangea domain.
+
+You'll need to set up and enable some rulesets in Redact service config.
```
Comment (kenany) [inline examples/store/README.md:15]: Is this actually necessary here?

File: examples/store/pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+[tool.poetry]
+name = "redact-examples"
```
Comment (kenany) [inline examples/store/pyproject.toml:2]:
```suggestion
name = "store-examples"
```

**Review by kenany — DISMISSED** [review]

**Review by kenany — DISMISSED** [review]

**Review by kenany — APPROVED** [review]

**Review by kenany — APPROVED** [review]

**Review by kenany — APPROVED** [review]

**Review by kenany — APPROVED** [review]

**Review by kenany — DISMISSED** [review]

**Review by kenany — APPROVED** [review]

File: examples/asyncio/share/poetry.lock [hunk 4]
```diff
@@ -918,7 +918,7 @@ requests-toolbelt = "^1.0.0"
 
 [package.source]
 type = "directory"
-url = "../../packages/pangea-sdk"
+url = "../../../packages/pangea-sdk"
```
Comment (kenany) [inline examples/asyncio/share/poetry.lock:921]: Huh, interesting that this was working at all?

**Review by kenany — DISMISSED** [review]

File: examples/asyncio/share/poetry.lock (hunk 4 above)
Comment (pangea-andrest) [reply examples/asyncio/share/poetry.lock:921]: It seemed to work locally with the `run_examples.sh` script, but asyncio examples were just pushed

**Review by kenany — DISMISSED** [review]

**Review by kenany — DISMISSED** [review]

**Review by kenany — APPROVED** [review]

**Comment by pangea-andrest** [comment]
Merge in https://github.com/pangeacyber/pangea-python/pull/317

**Pull request closed without merging**